trek.js

```diff
@@ -0,0 +1,81 @@
+const baseURL = 'https://trektravel.herokuapp.com/trips';
+
+$(document).ready(() => {
+  $('#reservationForm').hide();
+  $('#singleTripDetails').hide();
+  $('#displayArea').hide();
+  const loadAllTrips = () => {
+    $.get(baseURL, (response) => {
+      console.log('all trips worked');
+      console.log(response);
+      $('#displayArea').show();
+      response.forEach((trip) => {
+        const thisTrip = `<li id='${trip.id}'>${trip.name} </li>`;
+        $('#tripSection').append(thisTrip);
+      });
+    }).fail(() => {
+      console.log('Something went wrong with the load trips call');
+    });
+  };
+
+  const tripDetails = (tripId) => {
+    $.get(`${baseURL}/${tripId}`, (response) => {
+      console.log('Single trip worked');
+      console.log(response);
+      if ($('#details').length > 0) {
+        $('#details').remove();
+      }
+
+      if ($('#showForm').length > 0) {
+        $('#showForm').remove();
+      }
+      const details = `
+      <ul id='details' class='columns large-6 small-8'>
+        <li id='${response.id}'><strong>ID:</strong> ${response.id} </li>
+        <li><strong>Destination:</strong> ${response.name} </li>
```

Comment:
I like that you've wrapped these details in a `<ul>` rather than having (for example) a bunch of free-floating `<p>` tags.